README.md

```diff
@@ -1,3 +1,70 @@
 # DCTNA Helm Chart
 
+<!-- This README.md is generated. -->
+
 This repository contains the Helm Chart for installing and configuring [DCTNA](https://github.com/philips-labs/dct-notary-admin).
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+A Helm chart for DCTNA.
```

Comment:
Would be very helpfull to at least write the full name once for dctna

Reply:
Done updated in the template